pose_tracking:
  start_x: -43.829644
  start_y: -10.606556
  start_yaw: -0.4581

  # for self_dataset_1
  # 1:-43.829644  2:-27.888224 3: 192.829000
  # 1:-10.606556  2:-8.387609  3: -62.716500
  # 1:-0.4581     2:-2.8758    3: -2.7723

  # for self_dataset_2
  # 1:-7.021920
  # 1:-2.831730
  # 1:0.23442899

  # for kitti
  # for kitti-00 1: 1540->1650 2: 3360->3860 3: 4420->4540
  # 1: 102.255    2: 219.887    3: -6.69325
  # 1: 8.71256    2: -83.7607   3: 6.71705
  # 1: -3.0764523 2: 1.61859296 3: -1.08830784

  # for kitti-02 1: 4180->4280 2: 4549->4649
  # 1: 452.172    2: 838.594
  # 1: -506.687   2: -242.763
  # 1: 1.87993536 2: 1.08453647

  # for kitti-05 1: 1301->1581 2: 2300->2650
  # 1: 154.568     2: 8.24924
  # 1: -210.233    2: 15.5651
  # 1: 2.05878703  2: -1.48268205

  # for kitti-06 1: 834->1100
  # 1: -1.46266 the last pose and the incre_pose should be provided
  # 1: 0.0329893
  # 1: 0.0089339

  # for kitti-08 1: 1400->1850
  # 1: 156.38
  # 1: 209.722
  # 1: -3.08536179

  # for oxford 1: 500->8699 2: 14551->end
  # 1: -0.231070  2: 32.316244
  # 1: 3.752228   2: -111.111013
  # 1: 0.93545111 2: 2.53649271

  tracking_distance_threshold: 1.0 # according the experimental setup
  tracking_angle_threshold: 0.2 # according the experimental setup

  distance_association_vertex: 2.0 # 2.0
  distance_association_polygon: 3.0 # 3.0
  factor_vertex: 1.0
  factor_edge: 1.0

  corner_threshold: 10.0
  edge_threshold: 0.1
  corner_feature_num: 10
  edge_feature_num: 50
  edge_feature_selection: false
  constant_mode: true # false for Newer College dataset

  # change the output path
  outfile_pose: /home/ghm/map/self_dataset/info/pose_info.txt
  outfile_time: /home/ghm/map/self_dataset/info/pose_time.txt